base/bucket_gocb.go

```diff
@@ -1421,7 +1421,7 @@ func (bucket *CouchbaseBucketGoCB) GetXattr(k string, xattrKey string, xv interf
 		}
 
 	}
-	description := fmt.Sprintf("GetXattr %s", UD(k).Redact())
+	description := fmt.Sprintf("GetXattr %s", UD(k)().Redact())
```

Comment:
If we added a `Redact() string` method onto `RedactorFunc` that called the function, and then `Redact()` on the result, we wouldn't need to make these awkward looking changes in the code.